pgt/src/PGT.hs

```diff
@@ -137,7 +137,7 @@ runTasty tastyOptions postProcess config tests = liftIO $ do
         else System.exitFailure
 
 runTests :: MonadIO m => Config -> Tests -> m ()
-runTests config = runTasty mempty identity config . Vector.toList
+runTests config = runTasty mempty PGT.impureParse config . Vector.toList
```

Comment:
Comment: Instead of having the `pgt test` be linter enabled, I think it might be best to add a `--lint` flag to it. I could also do the same for the update command. Such that we can have `pgt test --lint` && `pgt update --lint`. 

OTWZ, I initially wanted to do this but it seemed a bit more work than adding new commands.

Reply:
__comment:__ @epicallan I do not have a strong opinion on this. I do think the less people need to do in order to ensure the format is correct, the better. Obviously I expect this to be enabled in CI, so ultimately it'll catch all formatting errors prior to merge, but the earlier we can catch problems the better.

In a perfect world, I'd probably have it lint by default and add a `--no-lint` flag *only* if I found there were cases where I wanted to skip linting. Although I cannot think of any cases offhand atm.

Reply:
@epicallan I'd probably only add the `--no-*` flags on the first use case. If there's never a need to disable something then I don't see the point in adding it until then.

Reply:
@dkubb Good point. Thanks